doc/man7/RAND_DRBG.pod

```diff
@@ -265,9 +265,9 @@ from the trusted entropy sources.
 =back
 
 NOTE: Manual reseeding is *not allowed* in FIPS mode, because
-NIST SP-800-90A mandates that entropy *shall not* be provided by the
-consuming application, neither for instantiation, nor for reseeding.
-[NIST SP 800-90Ar1, Sections 9.1 and 9.2]. For that reason the B<randomness>
+[NIST SP 800-90A Rev. 1] mandates that entropy *shall not* be provided by
+the consuming application, neither for instantiation (Section 9.1), nor for
```

Comment:
How about a colon after _application_ instead of a comma?
Or even a full stop.

Reply:
@paulidale I feel your uneasiness caused by all these commas. Probably the sentence is too much influenced by German grammar. However, splitting the sentence makes _me_ feel uneasy because the second part refers to the first.

As a compromise, I tried to find a formulation which sounds less awkward to a native speaker.